block_manipulation_service/src/test/java/com/stnetix/ariaddna/blockmanipulation/BMSTest.java

```diff
@@ -0,0 +1,76 @@
+package com.stnetix.ariaddna.blockmanipulation;
+
+import com.stnetix.ariaddna.vufs.businessobjects.Block;
+import com.stnetix.ariaddna.vufs.businessobjects.Metafile;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.junit.Assert.assertFalse;
+
+public class BMSTest {
+
+    private BlockGenerate blockGenerate;
+
+    @Before
+    public void before(){
+        if(blockGenerate == null) {
+            blockGenerate = new BlockGenerate();
+            blockGenerate.setLocalService(new LocalServiceStub());
+        }
+    }
+
+    @Test
+    public void getBlockSize() throws Exception {
+
+        String version="1";
+        String parentFileUuid = "1";
+        assertFalse(blockGenerate.getBlockSize()<=0);
+
+        //equal
+        Metafile metafileBelow = new Metafile(version,"3",parentFileUuid);
+        Block newBlock = blockGenerate.getNextBlock(metafileBelow);
+        assertFalse(newBlock.getSize()>=blockGenerate.getBlockSize());
+        try
+        {
+            newBlock = blockGenerate.getNextBlock(metafileBelow);
+            assertFalse(false);
```

Comment:
if you want make check on exception use: 
`@Test(expected = IllegalArgumentException.class)`